hw/ip/aes/pre_sca/alma/README.md

```diff
@@ -189,12 +189,14 @@ For more details, please refer to the [Alma tutorial](https://github.com/IAIK/co
 
    ```sh
    ./verify.py --json tmp/circuit.json \
-      --label tmp/labels.txt \
-      --top-module aes_sbox \
-      --vcd tmp/tmp.vcd \
-      --rst-name rst_ni --rst-phase 0 \
-      --probe-duration once --mode transient \
-      --glitch-behavior loose --cycles 6
+     --label tmp/labels.txt \
+     --top-module aes_sbox \
+     --vcd tmp/tmp.vcd \
+     --rst-name rst_ni --rst-phase 0 \
+     --probe-duration once \
+     --mode transient \
+     --glitch-behavior loose \
```

Comment:
I think we should remove loose glitch behavior (here and below) since this feauture may be buggy as Vedad explained once (actually he sent a mail saying that he fixed one severe bug related to that feature but I believe this is something else). Also we need to rerun the analysis.

Reply:
I was thinking about this, too. However, we are using slightly different tool versions/code bases for analyzing hardware accelerators (analyzed using Alma) and processors (Coco-Alma). I am not sure if that bug is also in the version we are using for AES/KMAC. I need to ask Vedad about this. Because switching to `--glitch-behavior strict` has a very high performance impact for the AES S-Box. The runtime increases from 45secs to 90min (which is roughly 100x). For KMAC it's less of an issue because the S-Box/non-linear layer there is much much simpler.

Anyway, the current RTL passes with both options :-)

Reply:
Cool!